release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -293,6 +307,22 @@ You can now set the maximum length of the syslog message in the Ingress Controll
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-controller-configuration-parameters_configuring-ingress[Ingress Controller configuration parameters].
 
+[id="ocp-4-10-openstack-dcn-for-enterprise"]
+==== Support for {rh-openstack} DCN (Technology Preview)
+
+You can now deploy a cluster on a {rh-openstack-first} deployment that uses a link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/distributed_compute_node_and_storage_deployment/index[distributed compute node (DCN)] configuration. There are a number of limitations:
+
+* Only {rh-openstack} version 16 is supported.
+* For {rh-openstack} 16.1.4, only hyper-converged infrastructure (HCI) and Ceph technologies are supported at the edge.
+* For {rh-openstack} 16.2, non-HCI and Ceph technologies are supported as well.
+* Networks must be created ahead of time (Bring Your Own Network) as either tenant or provider networks. These networks must be scheduled in the appropriate availability zones.
+
+[id="ocp-4-10-external-cloud-providers-openstack"]
+==== Support for external cloud providers for clusters on {rh-openstack} (Technology Preview)
+
+Clusters that run on {rh-openstack} now support external cloud providers. This capability is available as part of the `TechPreviewNoUpgrade` feature set.
```

Comment:
This sounds a bit misleading, it's not "external clouds providers", it's just an external cloud provider will be used instead of a legacy in-tree one. Basically [1] instead of [2], it's just the code got moved in K8s, but we kept using deprecated version of it.

[1] https://github.com/kubernetes/cloud-provider-openstack
[2] https://github.com/openshift/kubernetes/tree/master/staging/src/k8s.io/legacy-cloud-providers/openstack

Reply:
@dulek TY! So, an external cloud provider is being used instead of a legacy one. Is there a summary of user impact we could communicate here?

Reply:
Hm, in general legacy one was frozen, so we get some new features. I'd explicitly highlight an example of support for UDP Services of type `LoadBalancer`, but there's probably more that we haven't tested.

Reply:
Without more specifics, I can mention that. 

```suggestion
Clusters that run on {rh-openstack} can now use [Cloud Provider OpenStack](https://github.com/kubernetes/cloud-provider-openstack), which supports features like UDP services of the `LoadBalancer` type. 

This capability is available as part of the `TechPreviewNoUpgrade` feature set.
```

Thoughts? @dulek

Reply:
I'd be cautions about listing the new features that comes with the external cloud provider until they are all validated by QE.

Reply:
In that case:

```suggestion
Clusters that run on {rh-openstack} can now use link:https://github.com/kubernetes/cloud-provider-openstack[Cloud Provider OpenStack]. 
```